src/vm/coreassemblyspec.cpp

```diff
@@ -275,8 +275,23 @@ STDAPI BinderAcquirePEImage(LPCWSTR   wszAssemblyPath,
 
 STDAPI BinderHasNativeHeader(PEImage *pPEImage, BOOL* result)
```

Comment:
@gkhanna79 This is not exactly what you asked for, but I noticed that this function was missing the TRY/CATCH block that its peers had and was always returning S_OK.  I think this is a better fix, since it make this function correctly implement its implied contract.